.github/workflows/close-bug-report.yml

```
@@ -0,0 +1,34 @@
+name: Close release bug report issue when release branch gets merged
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+
+jobs:
+  close-bug-report:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.head.ref, 'Version-v')
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 1 # This retrieves only the latest commit.
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn --immutable
+
+      - name: Close release bug report issue
+        id: close-release-bug-report-issue
+        env:
+          BUG_REPORT_REPO: MetaMask-planning
+          BUG_REPORT_TOKEN: ${{ secrets.BUG_REPORT_TOKEN }}
+        run: npm run close-release-bug-report-issue
```

Nit: While it doesn't really matter, we may as well be consistent with using `yarn` to run scripts since that's the package manager we use.

```suggestion
        run: yarn run close-release-bug-report-issue
```

---

Makes sense, I also updated it for other Github actions where it was the case in this [commit](https://github.com/MetaMask/metamask-extension/pull/20391/commits/ae9f8d14eef7f7f5a2b5006d7d6d63ca0d0c9686)